app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md

```diff
@@ -51,7 +51,9 @@ Whilst school experience isn’t compulsory, it’s a good idea to visit a schoo
 
 ## 3.	Get a teacher training adviser
 
-Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. Whether it's just an idea, or you're looking to get back into the classroom, you could get help with things like finding school experience, choosing the right course for you, funding, and writing a successful application.
+All our teacher training advisers are experienced teachers who can provide you with additional support when preparing and applying for teacher training. You could get help with things like finding school experience, choosing the right course for you, funding, and writing a successful application.
+
+Chat by phone, text or email, as little or as often as you need.
 
 [Get a teacher training adviser](https://adviser-getintoteaching.education.gov.uk/).
```

Comment:
Do we want to take people to the TTA page here or deliberately to the funnel as they could be ETA candidates @EllieNodder ?

Reply:
hmm probably TTA page actually, if it's a blog about applying